src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,111 @@
+@use '../utils/variables' as *;
+@use '../utils/mixins' as *;
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+
+  width: $card-width;
```

Comment:
Ensure that the `$card-width` variable is defined in the `variables` file. If it's not defined, the card width might not be set correctly.